src/Files.App/Views/PaneHolderPage.xaml

```diff
@@ -70,84 +57,34 @@
 			NavParams="{x:Bind NavParamsLeft, Mode=OneWay}"
 			PaneHolder="{x:Bind}" />
 
-		<!--  Pane Right  -->
-		<Border
-			x:Name="PaneRightBorder"
-			Grid.Column="1"
-			x:Load="{x:Bind IsRightPaneVisible, Mode=OneWay}">
-			<shellpages:ModernShellPage
-				x:Name="PaneRight"
-				ContentChanged="Pane_ContentChanged"
-				Loaded="Pane_Loaded"
-				NavParams="{x:Bind NavParamsRight, Mode=OneWay}"
-				PaneHolder="{x:Bind}" />
-		</Border>
-
-		<!--  Pane Resizing Splitter  -->
+		<!--  Sizer  -->
 		<toolkit:GridSplitter
 			x:Name="PaneResizer"
 			Grid.Column="1"
-			Width="5"
-			MinWidth="5"
-			HorizontalAlignment="Left"
+			Width="4"
```

Comment:
I think width is already 4 by default, can you check?

Reply:
MinWidth yes. When I didn't set this prop explicity, it took 1.0* width.
![image](https://github.com/files-community/Files/assets/62196528/25682096-f93d-40a3-be16-364ad2548127)

Reply:
Okay